content/Video_Display.md

```diff
@@ -290,58 +290,59 @@ of this brightness correction.
 
 Writing to this register launches a DMA transfer from ROM or RAM to OAM
 memory (sprite attribute table). The written value specifies the
-transfer source address divided by 100h, that is, source & destination are:
+transfer source address divided by $100, that is, source and destination are:
 
 ```
-Source:      XX00-XX9F   ;XX in range from 00-F1h
-Destination: FE00-FE9F
+Source:      $XX00-$XX9F   ;XX = $00 to $F1
+Destination: $FE00-$FE9F
 ```
 
 The transfer takes 160 machine cycles: 152 microseconds in normal speed
 or 76 microseconds in CGB Double Speed Mode. On DMG, during this time,
-the CPU can access only HRAM (memory at FF80-FFFE); on CGB, the bus used
+the CPU can access only HRAM (memory at $FF80-$FFFE); on CGB, the bus used
 by the source area cannot be used (this isn't understood well at the
-moment, it's recommended to assume same behavior as DMG). For this
+moment; it's recommended to assume same behavior as DMG). For this
 reason, the programmer must copy a short procedure into HRAM, and use
 this procedure to start the transfer from inside HRAM, and wait until
 the transfer has finished:
 
 ```
  run_dma:
-  ld a, start address / 100h
-  ldh  (FF46h),a ;start DMA transfer (starts right after instruction)
-  ld  a,28h      ;delay...
+  ld a, start address / $100
+  ldh  [$FF46],a ;start DMA transfer (starts right after instruction)
+  ld  a,$28      ;delay...
  wait:           ;total 4x40 cycles, approx 160 μs
   dec a          ;1 cycle
   jr  nz,wait    ;3 cycles
   ret
 ```
 
 Because sprites are not displayed while OAM DMA is in progress, most
-programs execute this procedure from inside of their VBlank
-procedure. But it is also possible to execute it during display redraw
-also, allowing to display more than 40 sprites on the screen (that is, for
-example 40 sprites in upper half, and other 40 sprites in lower half of
-the screen), at the cost of a couple lines that lack sprites.
+programs execute this procedure from inside their V-Blank
+handler. But it is also possible to execute it during display redraw (Modes 2 and 3),
+allowing to display more than 40 sprites on the screen (that is, for
+example 40 sprites in the top half, and other 40 sprites in the bottom half of
+the screen), at the cost of a couple lines that lack sprites due to the fact that
+during those couple lines the PPU reads OAM as $FF.
```

Comment:
Apparently, glitches are possible if starting OAM DMA during Mode 3? (cc @hacktix) If we can't get conclusive evidence of the timing, we should mention that they are at least possible.

Reply:
Mode 2 and 3 specifically, confirmed again just today when I was adding a test ROM for exactly this to GBEmulatorShootout.

Reply:
Great! If the behavior is understood, could it be documented on Pan Docs, please?

Reply:
Done

Reply:
Well, it's not exactly understood. What's certain is that OAM reads don't always return $FF, as modifying the data that's transferred during either mode 2 or 3 can change attributes of the visual glitches. (I only managed to change the tile ID in my experiments through effectively brute-force so far, hoping to find a pattern to it eventually)

Reply:
Okay. @AlvaroBurnett's changes should be good enough for now, we'll just open a separate issue for this.